src/components/PartyFormSections/PartyInfoForm/PartyInfoForm.js

```diff
@@ -0,0 +1,74 @@
+import { FormattedMessage } from 'react-intl';
+import { Field } from 'react-final-form';
+
+import { Col, Row, Select, TextField } from '@folio/stripes/components';
+
+const PartyInfoForm = () => {
+  return (
+    <>
+      <Row>
+        <Col xs={3}>
+          <Field
+            component={Select}
+            dataOptions={[
```

Comment:
I'm not certain about these being pure Strings, but they are on the backend so they can remain for now. What you _should_ do though is translate the labels. Values can remain as is, but should probably be normalised (lower case, _ instead of spaces)